lhlist/src/ordered_set.rs

```diff
@@ -0,0 +1,103 @@
+use crate::Label;
+use crate::{Cons, False, LabeledValue, Member, Nil};
+
+impl OrderedHSet for Nil {}
+
+impl<H, T> OrderedHSet for Cons<LabeledValue<H>, T>
+where
+    H: Label,
+    T: OrderedHSet + Member<H, Output = False>,
+{}
+
+pub trait OrderedHSet: Sized {
```

Comment:
Does the set itself need to be `Sized` or does it just need to be `Sized` for the `prepend` method?

Reply:
It's only required for the `prepend` method - I have moved the trait bound on the method itself.